src/main/java/com/bobocode/petros/scaner/AnnotationDependencyConfigurationScanner.java

```diff
@@ -1,17 +1,110 @@
 package com.bobocode.petros.scaner;
 
+import com.bobocode.petros.annotation.ConfigClass;
+import com.bobocode.petros.annotation.Dependency;
 import com.bobocode.petros.container.DependencyDefinition;
+import com.bobocode.petros.exception.NoSuchPackageFoundException;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
 
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
-@SuppressWarnings("all")
+@Slf4j
 public class AnnotationDependencyConfigurationScanner implements DependencyScanner {
+    private final Map<String, List<DependencyDefinition>> configurationsMap;
+    @SuppressWarnings("FieldMayBeFinal")
+    private String targetClassesPath = "target/classes/";
+    private final String fileSeparator = System.getProperty("file.separator");
+
     public AnnotationDependencyConfigurationScanner() {
+        configurationsMap = new HashMap<>();
     }
 
+    @SneakyThrows
```

Comment:
Let`s avoid @SneakyThrows in our "production" code

Reply:
Done